Tests/LinuxMain.swift

```diff
@@ -68,6 +68,21 @@ extension HtmlCssInlinerTests {
     ("testHtmlCssInliner", testHtmlCssInliner)
   ]
 }
+extension InviteTests {
```

Comment:
dang, wonder if we should have a test phase that runs `make sourcery` and sees if `git status` outputs anything

Reply:
I wonder if it's possible to hook into build phases while running `swift generate-xcodeproj`.